k3ng_keyer/k3ng_keyer.ino

```diff
@@ -606,9 +606,38 @@ Recent Update History
   #define tone toneDUE
   #define noTone noToneDUE
 #elif defined(ARDUINO_MAPLE_MINI)
-  #include <SPI.h>
-  #include <Wire.h>
+//  #include <SPI.h>
+//  #include <Wire.h>
   #include <EEPROM.h>
+  void initialize_pins(void);
```

Comment:
What IDE are you using to compile?  You get an error if you don't have these declarations?